apps/frontend/src/utilities/delta_util.ts

```diff
@@ -155,9 +155,11 @@ export function get_eval_start_time(
   ev: ContextualizedEvaluation
 ): string | null {
   for (const prof of ev.contains) {
-    for (const ctrl of prof.contains) {
-      if (ctrl.hdf.segments!.length) {
-        return ctrl.hdf.segments![0].start_time;
+    if ('contains' in prof) {
```

Comment:
prof.contains is a array of contextualizedprofiles which themselves implement the contains interface but with item being an array of contextualizedcontrols so why is the type system throwing a fit for no good reason???????